activemodel/lib/active_model/attribute.rb

```diff
@@ -38,7 +38,7 @@ def initialize(name, value_before_type_cast, type, original_attribute = nil, val
       @value = value unless value.nil?
     end
 
-    def value
+    def value(&_)
```

Comment:
Could we not pass the block instead?

Reply:
I think this block is needed: https://github.com/rails/rails/blob/8a63fa558ffd0ad937f127059476df87dae7c61e/activemodel/test/cases/attribute_set_test.rb#L154-L158

That test actually still passes when this one takes no block, but there is also `fetch_value` from the `LazyAttributeSet` and when that is removed the test fails.